tensorboard/data/server/data_compat.rs

```diff
@@ -619,6 +650,76 @@ mod tests {
         }
     }
 
+    mod tensors {
+        use super::*;
+
+        #[test]
+        fn test_metadata_tensor_with_dataclass() {
+            let md = pb::SummaryMetadata {
+                plugin_data: Some(PluginData {
+                    plugin_name: "rando".to_string(),
+                    content: Bytes::from_static(b"preserved!"),
+                    ..Default::default()
+                }),
+                data_class: pb::DataClass::Tensor.into(),
+                ..Default::default()
+            };
```

Comment:
nit: We do have `blank_with_plugin_content` (per your suggestion), in
case you’d like to use that? Also suffices for the next test case, since
you can pass `DataClass::Unknown`.

Reply:
> (per your suggestion)

Done, lol.